src/task/CamelKTaskCompletionItemProvider.ts

```diff
@@ -22,65 +22,66 @@ import { TraitManager } from './TraitManager';
 
 export class CamelKTaskCompletionItemProvider implements vscode.CompletionItemProvider {
 
-    provideCompletionItems(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken, context: vscode.CompletionContext): vscode.ProviderResult<vscode.CompletionItem[] | vscode.CompletionList> {
-        return this.provideCompletionItemsForText(document.getText(), document.offsetAt(position));
-    }
+	provideCompletionItems(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken, context: vscode.CompletionContext): vscode.ProviderResult<vscode.CompletionItem[] | vscode.CompletionList> {
+		return this.provideCompletionItemsForText(document.getText(), document.offsetAt(position));
+	}
 
-    public async provideCompletionItemsForText(text: string, offset: number): Promise<vscode.CompletionItem[]> {
-        let node = jsonparser.findNodeAtOffset(jsonparser.parseTree(text), offset, false);
-        let completions: vscode.CompletionItem[] = [];
-        if (node !== undefined) {
-            if (this.isInTasksArray(node)) {
-                let completionBasic: vscode.CompletionItem = {
-                    label: 'Camel K basic development mode',
-                    insertText:
-`{
+	public async provideCompletionItemsForText(text: string, offset: number): Promise<vscode.CompletionItem[]> {
+		const globalNode = jsonparser.parseTree(text);
+		const node = jsonparser.findNodeAtOffset(globalNode, offset, false);
+		let completions: vscode.CompletionItem[] = [];
+		if (node !== undefined) {
```

Comment:
I think this line could be replaced by `if (node) {` which basically does the same check for undefined and also for empty string which is rather not relevant in this case.